src/__integrations__/pmClient.browser.docker.test.ts

```diff
@@ -46,8 +50,10 @@ describe('Data Submission Form', () => {
     await myBrowser.close();
   });
 
-  it('should call start session, when Continua is pressed', async () => {
+  it.only('should call start session, approve terms and add wallet when Continua is pressed', async () => {
```

Comment:
Why `only` ?

Reply:
@pasqualespica `only` is used to execute just one test case from a whole test suite. In this case is a typo